crates/eips/src/eip4844/mod.rs

```diff
@@ -87,8 +92,169 @@ pub const BYTES_PER_COMMITMENT: usize = 48;
 /// How many bytes are in a proof
 pub const BYTES_PER_PROOF: usize = 48;
 
-/// A Blob serialized as 0x-prefixed hex string
-pub type Blob = FixedBytes<BYTES_PER_BLOB>;
+#[derive(Clone, Debug)]
+pub struct Blob {
```

Comment:
we also need serde support.

deserialize we need to implement manually because we need an additional check that the decoded blob has the correct len